charts/cosigned/templates/webhook/service_monitor_webhook.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.serviceMonitor.enabled (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "cosigned.fullname" . }}-webhook-monitor
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "cosigned.labels" . | nindent 4 }}
+    control-plane: {{ template "cosigned.fullname" . }}-webhook
+spec:
+  endpoints:
+    - path: /metrics
```

Comment:
Any reason why we might want to parameterize the metrics path to a value?

Reply:
Why don't we want it ? It is the default path value and, as far as I remember, it is not configurable.

Reply:
I also guess that is not configurable, would suggest if that is configurable we can update this in a follow-up.

waiting your comment @sabre1041

Reply:
Lets get it in